perfdash/perfdash.go

```diff
@@ -134,37 +140,51 @@ func parseTestOutput(scanner *bufio.Scanner, buildNumber int, resources sets.Str
 	return hist
 }
 
-func getLatencyData(client *jenkins.JenkinsClient, job string) (TestToBuildData, sets.String, sets.String, error) {
+func getLatencyDataFormGCS() (TestToBuildData, sets.String, sets.String, error) {
+	fmt.Print("Getting Data from GCS...\n")
 	buildLatency := TestToBuildData{}
 	resources := sets.NewString()
 	methods := sets.NewString()
 
-	queue, err := client.GetJob(job)
+	buildNumber := *startFrom
+	latestBuildResponse, err := http.Get(fmt.Sprintf("%vlatest-build.txt", urlPrefix))
 	if err != nil {
-		return buildLatency, resources, methods, err
+		panic(err)
+	}
+	latestBuildBody := latestBuildResponse.Body
+	defer latestBuildBody.Close()
+	latestBuildBodyScanner := bufio.NewScanner(latestBuildBody)
+	latestBuildBodyScanner.Scan()
+	var lastBuildNo int
+	fmt.Sscanf(latestBuildBodyScanner.Text(), "BUILD_NUMBER=%d", &lastBuildNo)
```

Comment:
I feel like a bad person, since I just recommended using this file, but I'm thinking of changing the format so the file contains only the build number and nothing else (#21180). It would make this logic even cleaner, though, since you'd just need to read the content.